frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1267,7 +1274,7 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                               },
                           }
                         : null,
-                    featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_REPLAY]
+                    !conversionGoal && featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_REPLAY]
```

Comment:
TODO: Might check with Paul if there's an easy way to display recordings associated with a conversion